pkg/cmd/util/clientcmd/factory_client_access.go

```diff
@@ -267,25 +270,19 @@ func (f *ring0Factory) Printer(mapping *meta.RESTMapping, options kprinters.Prin
 			options.Kind = alias
 		}
 	}
-	return describe.NewHumanReadablePrinter(options), nil
+	return describe.NewHumanReadablePrinter(f.JSONEncoder(), f.Decoder(true), options), nil
 }
 
-func (f *ring0Factory) Pauser(info *resource.Info) (bool, error) {
+func (f *ring0Factory) Pauser(info *resource.Info) ([]byte, error) {
```

Comment:
@ncdc @mfojtik Please re-review: this drops the changes of https://github.com/openshift/origin/commit/b93b9cf3c9e3faf632ade9cbee6c0dc1d9d7e068. Also below for `Resumer`.